radio/src/lua/interface.cpp

```diff
@@ -981,7 +981,7 @@ class LuaWidgetFactory: public WidgetFactory
       }
 
       if (lua_pcall(L, 2, 1, 0) != 0) {
-        TRACE("Error in widget %s: %s", getName(), lua_tostring(L, -1));
+        TRACE("Error in create() of widget %s: %s", getName(), lua_tostring(L, -1));
```

Comment:
I am no expert :blush:, but it sounds weird. My suggestion: `Error in widget %s create() funciton: %s`